source/extensions/common/wasm/wasm.cc

```diff
@@ -2057,13 +2058,21 @@ WasmResult Context::getMetric(uint32_t metric_id, uint64_t* result_uint64_ptr) {
   return WasmResult::BadArgument;
 }
 
-Wasm::Wasm(absl::string_view vm, absl::string_view vm_id, absl::string_view vm_configuration,
+Wasm::Wasm(absl::string_view runtime, absl::string_view vm_id, absl::string_view vm_configuration,
            Stats::ScopeSharedPtr scope, Upstream::ClusterManager& cluster_manager,
            Event::Dispatcher& dispatcher)
-    : vm_id_(std::string(vm_id)), wasm_vm_(Common::Wasm::createWasmVm(vm)), scope_(scope),
-      cluster_manager_(cluster_manager), dispatcher_(dispatcher),
+    : vm_id_(std::string(vm_id)), wasm_vm_(Common::Wasm::createWasmVm(runtime, scope)),
+      scope_(scope), cluster_manager_(cluster_manager), dispatcher_(dispatcher),
       time_source_(dispatcher.timeSource()), vm_configuration_(vm_configuration),
-      stat_name_set_(scope_->symbolTable().makeSet("Wasm").release()) {}
+      wasm_stats_(WasmStats{ALL_WASM_STATS(
+          POOL_COUNTER_PREFIX(*scope_, std::string("wasm.") + std::string(runtime) + "."),
```

Comment:
`absl::StrCat` instead of `+` string

Reply:
Done here and in the other location with the same code.